calico/felix/futils.py

```diff
@@ -630,3 +632,35 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+
+def usage_count(calico_version, hostname, cluster_guid, cluster_size, cluster_type):
+    """Reports the cluster's guid, size and version to projectcalico.org.
+    Logs out of date calico versions, to the standard log file.
+
+    Keyword arguments:
+    calico_version -- the calico version
```

Comment:
There's a standard format for parameter docs, which is expected by the python docstring parser that we use:
```
:param <param name>: <param comment>
```